docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,242 @@
+no errors are introduced.
+
+Once the data is collected, it is often moved to a central storage
+machine and integrated with previously collected data. 
+**datashuttle** allows you to transfer project folders
+at the click of a button (graphical interface) or with a 
+single function call (Python API).
+
+Later on in the experiment, you may want to transfer only a subset
+of data from the central machine to separate computer for analysis. For example,
+you may  want to pilot an animal tracking pipeline, grabbing
+only the behavioural data for the first 5 subjects. 
+**datashuttle** allows flexible custom transfers, 
+meaning you don't have to drag and drop these data manually or
+write a custom script.
+
+**datashuttle** aims to drop into your existing acquisition pipelines whether 
+they be manual or automated, with two ways to use it:
+
+- The **graphical user interface** can be used instead of manual folder creation
+or transfer, reducing the risk of errors
+- The **Python API** can be used directly in automated acquisition
+pipelines, removing the need to write you own data-management code
+
+Below we will give an overview of **datashuttle**'s key folder creation
+and transfer features.
+
+## Creating folders with live-validation
+
+Creating folders through **datashuttle**'s graphical interface is as simple as
+entering the subject and session names, selecting the datatype and clicking 'create':
+
+
+```{image} /_static/blog_images/datashuttle/create-folders-example-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/create-folders-example-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Live-validation of inputs as you type ensures
+formatting errors don't creep into the project:
+
+```{image} /_static/blog_images/datashuttle/validation-bad-dark.png
+:align: center
+:class: only-dark
+:width: 500px
+```
+```{image} /_static/blog_images/datashuttle/validation-bad-light.png
+:align: center
+:class: only-light
+:width: 500px
+```
+<br>
+
+There are a number of shortcuts to ensure only
+information relevant to your specific project must be typed. 
+For example, the tags (`@DATE@`, `@TIME@`, `@DATETIME@`) will 
+fill the created folder  with the date / time / datetime. Double-clicking
+an input will suggest the next subject or session.
+A full list of shortcuts (e.g. automatic suggestion of next 
+subject / session names) are available in the 
+[documentation.](https://datashuttle.neuroinformatics.dev/pages/how_tos/create-folders.html#creating-project-folders)
+
+
+Folders can be created in an equivalent way through the Python API:
+
+```python
+from datashuttle import DataShuttle             # import datashuttle
+
+project = DataShuttle("my_first_project")       # initialise the project
+
+created_folder_paths = project.create_folders(  # create the folders
+    "sub-001", "ses-001_@DATE@", ["behav", "funcimg"]
+)
+```
```

Comment:
I don't think the comments add much, and we don't have them below.
```suggestion
```python
from datashuttle import DataShuttle

project = DataShuttle("my_first_project")

created_folder_paths = project.create_folders(
    "sub-001", "ses-001_@DATE@", ["behav", "funcimg"]
)
```
```